noxfile.py

```diff
@@ -543,7 +543,7 @@ def prerelease_deps(session):
 
     prerel_deps = [
         "protobuf",
-        "sqlalchemy<2.0.0",
+        "sqlalchemy>=1.4.16,<2.1",
```

Comment:
```suggestion
        "sqlalchemy>=1.4.16",
```

Reply:
Fixed in https://github.com/googleapis/python-bigquery-sqlalchemy/pull/1053/commits/dfbfcc32c048469f43616300acb18225d559ae63

I don't think we need the `>=` part since this will install the latest compatible version.